### DISCUSSION TITLE
Welcome to talos Discussions!

### DISCUSSION THREAD

**Discussion by andrewrynhard** [siderolabs/talos#2929] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NorthFace21** [comment]
Hi, I am not sure if I am at the right place! But here it goes. So I`ve been tasked by my organization to build up a Kubernetes Cluster in order to reduce some cloud cost. My current configuration would be 3 master nodes on vmware 7.0U1, vsphere 7.0d, and 4 worker node on physical Dell M630 nodes. They have an 80TB attached storage and I was hoping to be able (like vmfs a little bit) to be able to expose the same storage to all 4 worker in order for them to have access to data. (I believe cephfs is doing this is not some openfs or other)

I found TALOS along the way and I REALLY REALLY like the idea and the simplicity it seems to provide, however I must say that after almost 16 hours of brain digging information search and following all tutorial I could find I was not able to sucessfully initialize 1 control-plane. 

Here is a bit more information about my current setup:
I tried with alpha 0.10, ,beta 9.5 and stable 8.5 OVA image from the website. 
I follow the documentation and everything is working configuration store init.yaml and all everything is doing as it should, but EACH time I either do an INIT or a BOOTSTRAP I end up with a connection refused on the :6443 port. And I can`t figure out why its not completing the installation.


root@batman1:/home/adminrcgt/test2# talosctl   --talosconfig talosconfig  containers
NODE           NAMESPACE   ID         IMAGE            PID    STATUS
10.254.99.81   system      apid       talos/apid       3236   RUNNING
10.254.99.81   system      networkd   talos/networkd   3084   RUNNING
10.254.99.81   system      routerd    talos/routerd    3037   RUNNING
10.254.99.81   system      timed      talos/timed      3141   RUNNING
10.254.99.81   system      trustd     talos/trustd     3189   RUNNING

I am getting the based container running but it seems I can`t get the Kube-config or the control-plane running.

I am sure I am missing something VERY obvious, but right now I can't figured it out! I think this may be something with the configuration init.yaml or controlplane.yaml, missing the CNI for the network or something like this. I believe I would like to use cilium and rook/ceph for network and storage plugin, but I tougth I could do that setup after, since in no documentation it talk about configuring it at the begenning.

I`ve followed to the letter
https://www.talos.dev/docs/v0.8/virtualized-platforms/vmware/#create-the-machine-configuration-files

I really want this to work, I believe I could start the control plane on ubuntu directly but I really like the Talos idea at this time, something that will help my in the long term! 

If you need anything or if you have any recommendation I would be delighted!!!

THanks!

**Reply by andrewrynhard** [reply]
Hello, @NorthFace21 👋🏻 . Have you #3143 yet? I'd also like to invite you to our slack: http://slack.dev.talos-systems.io. We have a number of users on VMware that might be able to provide some insight.
